src/data/faq_de.json

```diff
@@ -42,9 +42,10 @@
-                            "Die Anzahl der Neuinfektionen wird 1x täglich vom RKI veröffentlicht. 'Gestern' bedeutet, dass in dem Wert alle gemeldeten Neuinfektionen vom Vortag bis 23:59 Uhr Uhr berücksichtigt wurden. ",
-                            "Die in der Corona-Warn-App angezeigten Statistiken werden täglich in 15-Minuten-Intervallen aus verschiedenen Quellen konsolidiert und entsprechend für die App aufbereitet. Die Statistiken zeigen neben aggregierten Werten vom Vortag (z.B. gemeldete Neuinfektionen) auch den R-Wert, der über ein sogenanntes „Nowcasting“ eine Schätzung für den aktuellen Tag darstellt.",
-                            "Das Konzept des R-Werts und die Datengrundlagen seiner Berechnung wurden detailliert im Epidemiologischen Bulletin des RKIs <a href='https://www.rki.de/DE/Content/Infekt/EpidBull/Archiv/2020/17/Art_02.html' target='_blank' rel='noopener'>(#17/2020 vom 23.04.2020)</a> beschrieben. Beispielrechnungen und die täglich aktualisierten R-Werte (4-Tages- und 7-Tages-R-Wert) stehen unter <a href='http://www.rki.de/covid-19-nowcasting' target='_blank' rel='noopener'>http://www.rki.de/covid-19-nowcasting</a> als Excel-Tabelle zur Verfügung."
+                            "Die Anzahl der Neuinfektionen wird 1x täglich vom RKI veröffentlicht. 'Gestern' bedeutet, dass in dem Wert alle gemeldeten Neuinfektionen vom Vortag bis 23:59 Uhr berücksichtigt wurden.",
+                            "Die weiterenin der Corona-Warn-App angezeigten Statistiken werden täglich in 15-Minuten-Intervallen aus verschiedenen Quellen konsolidiert und entsprechend für die App aufbereitet. Die Statistiken zeigen neben aggregierten Werten vom Vortag (z.B. gemeldete Neuinfektionen) auch den R-Wert, der über ein sogenanntes „Nowcasting“ eine Schätzung für den aktuellen Tag darstellt.",
+                            "Das Konzept des R-Werts und die Datengrundlagen seiner Berechnung wurden detailliert im Epidemiologischen Bulletin des RKIs <a href='https://www.rki.de/DE/Content/Infekt/EpidBull/Archiv/2020/17/Art_02.html' target='_blank' rel='noopener'>(#17/2020 vom 23.04.2020)</a> beschrieben. Beispielrechnungen und die täglich aktualisierten R-Werte (4-Tages- und 7-Tages-R-Wert) stehen unter <a href='http://www.rki.de/covid-19-nowcasting' target='_blank' rel='noopener'>http://www.rki.de/covid-19-nowcasting</a> als Excel-Tabelle zur Verfügung.",
+                            "Um die Datenqualität in den Statistiken der App sicherzustellen, werden alle Datenquellen auf Vollständigkeit und Konsistenz geprüft. Daher kann sich die Anzeige der validen Daten in der App im Vergleich zu anderen Webseiten/ Quellen zeitlich verzögern. Inhaltlich entsprechen die Werte den offiziellen Fallzahlen des Robert-Koch-Instituts."

```

Comment:
Webseiten/ Quellen => Webseiten/Quellen (remove space or use bzw. instead of / )

Robert-Koch-Instituts => Robert Koch-Instituts
Reference https://www.rki.de/ "Das Robert Koch-Institut ist ein Bundesinstitut im Geschäftsbereich des Bundesministeriums für Gesundheit"

Reply:
ok thanks